.eslintrc.json

```diff
@@ -12,13 +12,13 @@
     "ecmaVersion": 2018,
     "sourceType": "module"
   },
-  "extends": ["eslint:recommended", "plugin:react/recommended", "airbnb"],
+  "extends": ["next", "next/core-web-vitals", "plugin:prettier/recommended"],
   "settings": {
     "react": {
       "version": "detect"
     }
   },
-  "plugins": ["react", "jsx-a11y", "import"],
+  "plugins": ["prettier"],
```

Comment:
I think if we have "plugin:prettier/recommended" in extends, there's no need to use plugins at all, wdyt? https://www.npmjs.com/package/eslint-plugin-prettier/v/2.7.0#recommended-configuration